npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -128,6 +148,11 @@ func isBaseChain(chain string) bool {
 func (pMgr *PolicyManager) bootup(_ []string) error {
 	klog.Infof("booting up iptables Azure chains")
 
+	// Stop reconciling so we don't centend for iptables, and so we don't update the staleChains at the same time as reconcile()
+	// Reconciling would only be happening if this function were called to reset iptables well into the azure-npm pod lifecycle.
+	pMgr.reconcileManager.forceLock()
+	defer pMgr.reconcileManager.forceUnlock()
+
 	// 1. delete the deprecated jump to AZURE-NPM
 	deprecatedErrCode, deprecatedErr := pMgr.runIPTablesCommand(util.IptablesDeletionFlag, deprecatedJumpFromForwardToAzureChainArgs...)
```

Comment:
Do we want it locked for both `pMgr.runIPTablesCommand` and `pMgr.allCurrentAzureChains()`? If not we can just wrap the portion with lock/unlock without using defer

Reply:
Ya we'd like to for runIPTables, allCurrentChains, and restore since they all use iptables. We want to lock for this whole period so we don't start using iptables in the other go routine